docs/conf.py

```diff
@@ -122,7 +122,10 @@
     'docroles'
 ]
 
-autodoc_default_flags = ['show-inheritance', 'members']
+autodoc_default_options = {
```

Comment:
This look like a unreleated changes. This should be done in a separate PR.

Reply:
yeah, I was trying to figure out why the doc step was failing in the build on travis